docs/DeveloperGuide.adoc

```diff
@@ -294,36 +295,55 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+| `* * *` | student studying multiple subjects | categorize my flashcards under different modules | study them in such manner
+| `* * *` | meticulous student | sort my subjects manner.  | study subjects in an orderly
+| `* * *` | meticulous student | make use of colour codes  | highlight and know my weaknesses
+| `* * *` | advanced user | search through my content   | find subjects/topics easily
+| `* * *` | meticulous student,   | use the calendar to plan my study |  so that I will not miss any subject
+| `* * *` | meticulous student | view what tasks are overdue | know what I have not done
+| `* * *` | meticulous student | a mock test timer to simulate exam situation | be prepared for my upcoming exams
+| `* * *` | meticulous student | label difficult questions | the flash cards will appear more frequently for better knowledge absorption
+| `* * *` | student | move backward through the flashcards  | recall the previous question/answer
+| `* *`  | meticulous student | view the overall statistic | monitor my performance overtime
+| `* *`  | meticulous student | use the scoring system  | know how well prepared I am
+| `* *`  | As a user | hide the answer until I finish all the questions | remember better
+| `* *`  | meticulous student | check the correctness of my answer | know whether my answer was right
+| `* *`  | busy student | use the night mode  | study in a dark environment
+| `* *`  | student with many friends | export flashcards in a simple shareable format | share with my friends
+| `* *`  | artistic student | customise the theme | so that I would have a better user experience
+| `* *`  | colour blind student | special accessibility options  | see my flashcards clearly
+| `*` | competitive student | export and share my statistics with my friends | compare my performance to theirs
+| `*` | forgetful student | key in my answer | see my own attempt to compare with the correct answer
+| `*` | competitive student | compare my results with others | so that I can be recognised for my skills and knowledge
 
-|`* * *` |user |delete a person |remove entries that I no longer need
-
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is `KeyboardFlashCards` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
 
 *MSS*
```

Comment:
The title of this use case is absent.

Reply:
You can consider adding UC identifiers in case you need to refer to them in future.